core/kernel/api/class.ModelExporter.php

```diff
@@ -29,29 +29,33 @@ class core_kernel_api_ModelExporter
     /**
      * Export the entire ontology
      *
+     * @param string $format (optional) which format resulted ontology will be.
+     *
      * @return string
      */
-    public static function exportAll()
+    public static function exportAll($format = 'rdfxml')
```

Comment:
Just a suggestion: maybe move 'rdfxml' to a constant?

Reply:
good catch. Will take a look.